spec/parts/linux/cloud-init/artifacts/localdns_spec.sh

```diff
@@ -39,6 +41,8 @@ EOF
 cat <<EOF > "$RESOLV_CONF"
 nameserver 10.0.0.1
 nameserver 10.0.0.2
+nameserver 10.0.0.3
+nameserver 10.0.0.4
```

Comment:
Just a FYI, this test case does not cover the case Patrick mentioned previously. 
Patrick meant (even though it will only happens in pod resolv.conf, it will not happen here) the 4th+ nameserver will be commented out in the pod resolv.conf.

But you don't need to change this, because that case will never happen on node.

Reply:
It didn't test the /^nameserver/ change though. But as long as the comment does not happen, it is fine